topo/node/alpine/alpine.go

```diff
@@ -38,6 +39,19 @@ func New(nodeImpl *node.Impl) (node.Node, error) {
 	n := &Node{
 		Impl: nodeImpl,
 	}
+	idx := 1
```

Comment:
this might make sense to move to its own func

Reply:
done